Doc/library/random.rst

```diff
@@ -364,6 +364,11 @@ Sometimes it is useful to be able to reproduce the sequences given by a
 pseudo-random number generator.  By re-using a seed value, the same sequence should be
 reproducible from run to run as long as multiple threads are not running.
 
+Hash randomization, which is enabled by default since version 3.3, is not affected by
```

Comment:
```suggestion
Hash randomization is not affected by
```
Hash randomization is now the default in all currently supported versions of Python so this detail may not be needed here.